woocommerce/woocommerce-android
# Quantity rules analytics

Closes: #8620

### Why
We are adding read-only support for the Min / Max quantity extension

### Description
This PR adds analytics tracks to the usage of the Min / Max quantity extension read-only support within the app.
* `product_detail_view_quantity_rules_tapped` [event registration](https://github.com/Automattic/tracks-events-registration/pull/1518)
* `product_variation_view_quantity_rules_tapped` [event registration](https://github.com/Automattic/tracks-events-registration/pull/1519)

### Testing instructions
TC1
1. Open the Products List Screen
2. Open an order containing quantity rules
3. Check that the quantity rules card information is shown
4. Tap on the quantity rules card
5. Check that quantity rules details are shown
6. Check that the `product_detail_view_quantity_rules_tapped` event is sent

TC2
1. Open the Products List Screen
2. Open a variable product containing a variation with quantity rules
3. Tap on Variations
4. Tap on the variation containing quantity rules
5. Check that the quantity rules card information is shown
6. Tap on the quantity rules card
7. Check that quantity rules details are shown
8. Check that the `product_variation_view_quantity_rules_tapped` event is sent

### Images/gif
```
🔵 Tracked: product_detail_view_quantity_rules_tapped, Properties: {"blog_id":214253715,"is_wpcom_store":true,"is_debug":true}

🔵 Tracked: product_variation_view_quantity_rules_tapped, Properties: {"blog_id":214253715,"is_wpcom_store":true,"is_debug":true}
```
